src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -322,20 +322,22 @@ cleared.
+| 0  | An attacker can use all the threads of the service's thread pool, requesting for a time-consuming operation on each one of them, to leave the less amount of free time possible to other clients.                                                                                                                                                                                                                                                                            | The requests take more time to execute.                                                                                    |
+| 2  | The authentication token is stored with confidentiality by the clients. If it is stolen, the service cannot prevent access with it unless the token is revoked.                                                                                                                                                                                                                                                                                                              | All the client's keys can be used, and exported if allowed by the key's policy.                                            |
+| 3  | Parsec dependencies are not checked for Security Vulnerabilities.                                                                                                                                                                                                                                                                                                                                                                                                            | A vulnerability in one of Parsec dependency will also impact Parsec and the data Parsec shares with that dependency.       |
+| 4  | While Parsec is authenticated on the device, anyone can brute force a key ID to execute operations with the key. Unmitigated for the PKCS 11 Provider: all sessions share the login state; if one session logs in then all other opened sessions will also be logged in. Other sessions only need a valid key ID to execute operations with private keys. Mitigated for the TPM Provider: each key is protected by a long, random authentication value, generated by the TPM | All the client's PKCS 11 keys can be used with PKCS 11 operations.                                                         |
+| 5  | (DSPC) Parsec does not know about the security quality of a client's Unix password.                                                                                                                                                                                                                                                                                                                                                                                          | All the client's keys can be used, and exported if allowed by the key's policy.                                            |
```

Comment:
Can you make the "conditional" stuff more consistent - i.e. the authenticator "code" should be italic and it should either always be at the beginning or at the end. At the beginning might be better, so you know as soon as you start reading.